packages/create-v2-addon-repo/src/blueprints/test-app/config/ember-try.js

```diff
@@ -8,21 +8,28 @@ module.exports = async function () {
   return {
     usePnpm: true,
     scenarios: [
-      {
-        name: 'ember-lts-3.28',
-        npm: {
-          devDependencies: {
-            '@ember/test-helpers': '2.9.4',
-            '@types/ember__test-helpers': '2.9.1',
-            '@types/ember-qunit': '6.1.1',
-            'ember-a11y-testing': '5.2.1',
-            'ember-cli': '~4.12.3',
-            'ember-qunit': '6.0.0',
-            'ember-resolver': '11.0.1',
-            'ember-source': '~3.28.12',
-          },
-        },
-      },
+      /*
+        If your v2 addons need to support `ember-source@3.28`,
+        please update your `blueprints-v2-addon` to use
+        `@glimmer/component@1.1.2`.
+
+        Otherwise, you can delete this scenario.
+      */
+      // {
+      //   name: 'ember-lts-3.28',
+      //   npm: {
+      //     devDependencies: {
+      //       '@ember/test-helpers': '2.9.4',
+      //       '@types/ember__test-helpers': '2.9.1',
+      //       '@types/ember-qunit': '6.1.1',
+      //       'ember-a11y-testing': '5.2.1',
+      //       'ember-cli': '~3.28.0',
+      //       'ember-qunit': '6.0.0',
+      //       'ember-resolver': '11.0.1',
+      //       'ember-source': '~3.28.0',
+      //     },
+      //   },
+      // },
```

Comment:
[`@glimmer/component@2.0.0` dropped support for `ember-source<4.10`](https://github.com/emberjs/ember.js/blob/main/packages/%40glimmer/component/CHANGELOG.md) so this `ember-try` scenario fails out of the box if we try to create a Glimmer component (w/ an empty class and template) in a v2 addon.

```sh
Could not find module `@ember/owner` imported from `my-addon`
```